### ISSUE TITLE
Intel on submerged vs emerged water units

### ISSUE THREAD

**Issue by EvildrewFAF** [FAForever/fa#4423]
Radar cannot penetrate water so if radar cannot spot subs below water, why can subs with radar under water detect boats, planes, etc above water using radar in FAF? It makes little sense. If possible, it would be nice to have the Radar deactivated or set to 0 when a navy unit is submerged, and also the Vision radius reduced. This would be particularly noticeable on the Atlantis, tempest f.ex. as it acts like a perimeter whether emerged or submerged and in a fight vs cybran with stealthed navy it should change the dynamics.
If this feature is possible to be coded it could be also carried further to reshape the dynamic of T1-T3 submarines by altering their intel. That may make subs more useful or not in the context of not being spotted so easily, I will have to think about the values to make the case. One other aspect of reshaping this dynamic would also include making sonars have a smaller sonar radius and giving torpedo bombers a radar and possibly an even small sonar radius tbd.

So in short the question is, can someone tell me if it is possible to code this feature, maybe depending on elevation activate/set the radar radius and increase vision radius to new values and reverse the values when submerging?

**Comment by EvildrewFAF** [comment]
Postponed,...
